README.md

```diff
@@ -107,7 +106,54 @@ dependency on *ring*.
 Rustls requires Rust 1.61 or later.
 
 [ring-target-platforms]: https://github.com/briansmith/ring/blob/2e8363b433fa3b3962c877d9ed2e9145612f3160/include/ring-core/target.h#L18-L64
-[crypto::CryptoProvider]: https://docs.rs/rustls/latest/rustls/crypto/trait.CryptoProvider.html
+[`crypto::CryptoProvider`]: https://docs.rs/rustls/latest/rustls/crypto/trait.CryptoProvider.html
+[`ring`]: https://crates.io/crates/ring
+
+### Cryptography Providers
```

Comment:
These new sections use capitals on every word, which existing sections do not. I (weakly) prefer the latter style.

Reply:
Fixed the new headings (and the pre-existing  "Design Overview" heading)